includes/class-maera-restaurant-styles.php

```diff
@@ -341,67 +270,25 @@ function color_calculations() {
 
 			$style = '';
 
-			$style .= 'body a{';
-			$style .= 'color: #'. $content_links .';';
-			$style .= '}';
-
-			$style .= 'body a:hover{';
-			$style .= 'color: #'. $content_primary .';';
-			$style .= '}';
-
-			$style .= 'body, body h1, body h2, body h3, body h4, body h5, body h6, body p{';
-			$style .= 'color: '. $body_font_color .';';
-			$style .= '}';
-
-			$style .= '#wrap-main-section h1, #wrap-main-section h2, #wrap-main-section h3, #wrap-main-section h4, #wrap-main-section h5, #wrap-main-section h6, #wrap-main-section p{';
-			$style .= 'color: '. $content_font_color .';';
-			$style .= '}';
-
-			$style .= '.label-primary{';
-			$style .= 'background-color: #'. $content_primary .';';
-			$style .= '}';
-
-			$style .= '.navbar-default{';
-			$style .= 'background-color: #'. $content_dark .';';
-			$style .= '}';
-
-			$style .= '.navbar-default .navbar-nav > li > a {';
-			$style .= 'color: #'. $dark_font .';';
-			$style .= '}';
-
-			$style .= '#section_1 h1, #section_1 h2, #section_1 h3, #section_1 h4, #section_1 h5, #section_1 h6, #section_1 p{';
-			$style .= 'color: '. $section1_font_color .';';
-			$style .= '}';
-
-			$style .= '#section_2 h1, #section_2 h2, #section_2 h3, #section_2 h4, #section_2 h5, #section_2 h6, #section_2 p{';
-			$style .= 'color: '. $section2_font_color .';';
-			$style .= '}';
-
-			$style .= '#section_3 h1, #section_3 h2, #section_3 h3, #section_3 h4, #section_3 h5, #section_3 h6, #section_3 p{';
-			$style .= 'color: '. $section3_font_color .';';
-			$style .= '}';
-
-			$style .= '#section_4 h1, #section_4 h2, #section_4 h3, #section_4 h4, #section_4 h5, #section_4 h6, #section_4 p{';
-			$style .= 'color: '. $section4_font_color .';';
-			$style .= '}';
-
-			$style .= '#section_5 h1, #section_5 h2, #section_5 h3, #section_5 h4, #section_5 h5, #section_5 h6, #section_5 p{';
-			$style .= 'color: '. $section5_font_color .';';
-			$style .= '}';
-
-			$style .= '#footer{';
-			$style .= 'background-color: #'. $content_dark .';';
-			$style .= 'color: #'. $dark_font .'';
-			$style .= '}';
-
-			$style .= '#footer p{';
-			$style .= 'color: #'. $dark_font .'';
-			$style .= '}';
+			$style .= 'body a{color:#' . str_replace( '#', '', $content_links ) . ';}';
```

Comment:
if a color is prefixed with #, make sure its value does not have a # by using `str_replace( '#', '', $color )`